src/model.py

```diff
@@ -238,6 +263,22 @@ async def generate(self, request):
                 stream = stream.as_numpy()[0]
             else:
                 stream = False
+            exclude_input_in_output = pb_utils.get_input_tensor_by_name(
+                request, "exclude_input_in_output"
+            )
+            if exclude_input_in_output:
+                exclude_input_in_output = exclude_input_in_output.as_numpy()[0]
+            elif exclude_input_in_output is None and stream:
+                exclude_input_in_output = True
+            else:
+                exclude_input_in_output = False
+
+            if not exclude_input_in_output and stream:
+                exclude_input_in_output = True
```

Comment:
I think I would error here instead of switching it -

Reply:
To clarify my understanding - we'll pick the correct default if this flag is left unspecified (optional) - but we'll error check it if specified?

Reply:
> we'll pick the correct default if this flag is left unspecified (optional)

yes

> but we'll error check it if specified

at this point the error is only possible, if `exclude_input_in_output` was provided by user and `stream` was provided by user.